### ISSUE TITLE
Add `params_to_tune` for new NNs (`RNNModel`, `MLPModel`)

### ISSUE THREAD

**Issue by Mr-Geekman** [tinkoff-ai/etna#1196]
### 🚀 Feature Request

Add `params_to_tune` for new NNs (rnn, mlp).

### Proposal

Sources:
- pycaret
  - [mlp](https://github.com/pycaret/pycaret/blob/master/pycaret/containers/models/regression.py#L1482)
- kats
  - [lstm](https://github.com/facebookresearch/Kats/blob/992b95e1a8c34402756ffea2376189568de85a2c/kats/models/lstm.py#L296)

Suggested grid for `RNNModel`:
```
{
    "num_layers": IntUniformDistribution(1, 3, step=1),
    "hidden_size": IntUniformDistribution(4, 64, step=4),
    "lr": LogUniformDistribution(1e-5, 1e-2),
}
```

Suggested grid for `MLPModel`:
```
{
    "num_layers": IntUniformDistribution(1, 3, step=1),
    "hidden_size": IntUniformDistribution(4, 64, step=4),
    "lr": LogUniformDistribution(1e-5, 1e-2),
}
```

To discuss:
- Is tuning `lr` necessary?
- It is unclear how to parameterize `MLPModel` parameter `hidden_size` because it accepts list of sizes for the layers. Even if we come up with parametrization it is unclear how to implement it because it is expected that we have a complete set of parameter values. We can use `CategoricalDistribution` but it seems not very effective.

### Test cases

Look at https://github.com/tinkoff-ai/etna/issues/1184.

**Comment by martins0n** [comment]
* Is tuning lr necessary? - I think it's ok for now but may be can make lr a little bit wider from the left
* we can add context size for rnn

**Comment by Mr-Geekman** [comment]
Closed by #1218.
